cinder/Dockerfile

```diff
@@ -1,35 +1,25 @@
-# Cinder driver container for Dell EMC VNX and Unity
+# Cinder driver container for Dell Unity
 # Using openstack-cinder-volume base image
 
-FROM registry.redhat.io/rhosp-rhel8/openstack-cinder-volume:16.2.4
+FROM registry.redhat.io/rhosp-rhel9/openstack-cinder-volume:17.1
 
 ARG storops_version=1.2.11
 
-LABEL maintainer="Dell EMC" \
-      description="Red Hat OpenStack Platform 16.2 cinder-volume Dell EMC VNX and Unity" \
-      summary="Red Hat OpenStack Platform 16.2 cinder-volume Dell EMC VNX and Unity" \
-      name="dellemc/openstack-cinder-volume-dellemc-rhosp16" \
-      vendor="Dell EMC" \
-      version="4.4.1" \
-      storops=${storops_version} \
-      naviseccli="7.33.9.1.55"
```

Comment:
Is this not needed?

Reply:
The RHOSP17 is based on rhel9, the so libs naviseccli needs are for rhel8. Considering we don't have resource to check and make naviseccli work on rhel9 and the customer's requirement is Unity. So drop it first unless there is requirement.